src/applications/vaos/components/layout/CCLayout.jsx

```diff
@@ -108,14 +108,14 @@ export default function CCLayout({ data: appointment }) {
             APPOINTMENT_STATUS.cancelled === status) && (
             <Prepare>
               <p className="vads-u-margin-top--0 vads-u-margin-bottom--0">
```

Comment:
Since this block is repeated 6 times in these different layout files, you might consider making it a "shared component" so that future updates (if any) would need to be done only once.

Reply:
Though the blocks have the same text content, some are `<p>` elements and others are `<li>` at which point it seems like the small number of duplications is not worth creating additional shared components for. If we repeat this more in the future I think we'll update the `<Prepare>` element to accommodate.